lib/ansible/modules/monitoring/zabbix/zabbix_host_triggers.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) stephane.travassac@fr.clara.net
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+output:
+    description: Host Zabbix Triggers in Json format
+    returned: On success
+    type: complex
+    contains:
+            triggers_ok:
+                - comments: Raise an alert whenever there is no data received from the agent during
+                    the given interval
+                  description: 'Status {HOSTNAME}: server is unreachable'
+                  error: ''
+                  expression: "{13665}=1"
+                  flags: '0'
+                  lastchange: '1556112352'
+                  priority: '4'
+                  state: '0'
+                  status: '0'
+                  templateid: '13619'
+                  triggerid: '14038'
+                  type: '0'
+                  url: ''
+                  value: '0'
+            triggers_problem:
+                - comments: ''
+                  description: 'Process: NTPd is down'
+                  error: ''
+                  expression: "{13667}=0"
+                  flags: '0'
+                  last_event:
+                    acknowledged: '0'
+                    acknowledges:
+                    - alias: Admin
+                      clock: '1556182269'
+                      message: waiting customer action
+                    clock: '1556182854'
+                    eventid: '630'
+                    value: '1'
+                  lastchange: '1556182854'
+                  priority: '4'
+                  state: '0'
+                  status: '0'
+                  templateid: '13615'
+                  triggerid: '14040'
+                  type: '0'
+                  url: ''
+                  value: '1'
+'''
+
+DOCUMENTATION = '''
+---
+module: zabbix_host_triggers
+short_description: Get all triggers about a Zabbix host
+description:
+   - This module allows you to see if a Zabbix host have no active alert to make actions on it.
+     For this case use module Ansible 'fail' to exclude host in trouble.
+   - Length of "triggers_ok" allow if template's triggers exist for Zabbix Host
+version_added: "2.9"
+author:
+    - "Stéphane Travassac (@stravassac)"
+requirements:
+    - "python >= 2.7"
+    - zabbix-api
+options:
+    server_url:
```

Comment:
There is a lot of duplicate options that are already present in [this document fragment](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/zabbix.py). All of the listed options in the document fragemnt are required in Zabbix modules. Please, remove them from here and add missing options to instance created at line 198. Module [zabbix_host_facts](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py) is a good example for this.